telosnetwork/open-block-explorer
# #440 | Chain switcher updates

# Fixes #440 

## Description

This PR updates the appearance of the chain icons in the chain switcher sidebar

## Test scenarios

- this must be tested locally to enable the sidebar
- in your `.env` file, uncomment the line `#SHOW_SIDEBAR=true`
- run `yarn dev`
- go to http://localhost:8080
- note that the changes have been made from the issue

## Checklist:
